src/HPRT/toolkit/views.py

```diff
@@ -330,43 +331,72 @@ class HopkinsPart2DetailView(LoginRequiredMixin, DetailView):
     template_name = 'hp2_detail.html'
 
 
-def make_pdf(canvas, info, questions):
-    textobject = canvas.beginText()
-    textobject.setTextOrigin(inch, 10*inch)
-    textobject.setFillColorRGB(0.4,0,1)
-    textobject.setFont("Helvetica-Oblique", 60)
-    textobject.textLines("HPRT")
-    textobject.setFont("Times-Bold", 13)
-    textobject.textLines("")
-    textobject.setFillColor(colors.black)
-    textobject.textLines("Patient Name: " + info["patient"])
-    textobject.textLines("Doctor Administered: Dr. " + info["doctor"])
-    textobject.textLines("Date Administered: " + info["date"])
-
-    para_questions = []
+# def make_pdf(canvas, info, questions):
+#     textobject = canvas.beginText()
+#     textobject.setTextOrigin(inch, 10*inch)
+#     textobject.setFillColorRGB(0.4,0,1)
+#     textobject.setFont("Helvetica-Oblique", 60)
+#     textobject.textLines("HPRT")
+#     textobject.setFont("Times-Bold", 13)
+#     textobject.setFillColor(colors.black)
+#     textobject.textLines("Patient Name: " + info["patient"])
+#     textobject.textLines("Doctor Administered: Dr. " + info["doctor"])
+#     textobject.textLines("Date Administered: " + info["date"])
+
+#     para_questions = []
+#     styles = getSampleStyleSheet()
+
+#     for [question, answer] in questions:
+#         para_questions.append([Paragraph(question, styles['Normal']), Paragraph(str(answer), styles['Normal'])])
+
+#     if "score" in info:
+#         scoretext = canvas.beginText()
+#         scoretext.setTextOrigin(inch, (11*inch - (0.4*inch*len(questions))))
+#         scoretext.setFillColor(colors.black)
+#         scoretext.textLines("Total Score: " + info["score"])
+
+#     table = Table(para_questions, colWidths=3*inch, rowHeights=None, repeatRows=0)
+#     table.setStyle(TableStyle([
+#          ('INNERGRID', (0,0), (-1,-1), 0.4, colors.black),
+#          ('BOX', (0,0), (-1,-1), 0.5, colors.black),
+#          ]))
+
+#     canvas.drawText(textobject)
+#     table.wrapOn(canvas, 8.5*inch, 11*inch)
+#     table.drawOn(canvas, 1*inch, (12*inch - (0.4*inch*len(questions))))
+#     if "score" in info:
+#         canvas.drawText(scoretext)
+
+def make_pdf(response, info, questions):
```

Comment:
Can you add some sort of comment just describing in maybe a sentence what this function does?  You could also add just a brief thing about the parameters it takes and if it were to have a return value, a description of that as well. I know we haven't done much documentation in general, but I think it will help us out a lot if we can slowly populate the code base with decent comments, so we can help both ourselves and anyone else who does work on this project at some point.